### ISSUE TITLE
In the statsd metrics, blank spaces must be sanitized (according to etsy statsd format)

### ISSUE THREAD

**Issue by rudigerlove** [F5Networks/f5-telemetry-streaming#184]
### Environment
 * Telemetry Streaming Version:
 * BIG-IP Version:

### Summary
Please also include information about the reproducibility and the severity/impact of the issue.

### Expected Behavior
In our lab, we have some metrics that are coming in with spaces in the metric names and this is not correct. (ex: ARP flood stats)
I expect that there is some sanitization that is performed.
The F5 telemetry plugin must sanitize the metric names and remove spaces if any, like its mentioned here:
https://github.com/statsd/statsd/blob/db4ec7b16b5d98b5b797cdb5892d8e2d25cbbc74/stats.js#L168-L176
```
function sanitizeKeyName(key) {
  if (keyNameSanitize) {
    return key.replace(/\s+/g, '_')
              .replace(/\//g, '-')
              .replace(/[^a-zA-Z_\-0-9\.]/g, '');
  } else {
    return key;
  }
}
```

### Actual Behavior
A clear and concise description of what actually happens.
Please include any applicable error output.

f5telemetry.server-CVv15d01-ve-service-63kgfh-ad-testbed-dcn.dos-DeviceStats.ARP flood stats.common-aggDetected:0|g|#common.status:Ready,deviceType:Static Vector - Inline

**Comment by dstokesf5** [comment]
Thank you for your feedback. I have added this issue to our internal product backlog as AUTOTOOL-2938.

**Comment by dstokesf5** [comment]
This has been resolved in the 1.27.0 release.
